zipline/pipeline/engine.py

```diff
@@ -273,6 +276,12 @@ def _inputs_for_term(self, term, workspace, graph):
             out.append(input_data)
         return out
 
+    def get_loader(self, term):
+        if term is AssetExists():
```

Comment:
Do we ever actually hit this branch?  It doesn't look like we're handling a `None` return anywhere.